torch/_inductor/compile_fx.py

```diff
@@ -1037,7 +1037,7 @@ def cudagraphify_impl(
     Assumes inputs[static_input_idxs[i]] are always the same memory address
     """
     check_input_idxs = get_input_idxs_to_check(inputs, static_input_idxs)  # type: ignore[arg-type]
-    static_input_idxs = remove_unaligned_input_idxs(inputs, static_input_idxs)  # type: ignore[arg-type]
+    static_input_idxs = set(remove_unaligned_input_idxs(inputs, static_input_idxs))  # type: ignore[arg-type]
```

Comment:
Please use `OrderedSet` for deterministic behavior

Reply:
@oulgen if we have blanket banned regular set from inductor, even if it's used in a way that wouldn't cause nondeterminism, we should have a lint for it.

Reply:
@eellison was writing a linter, not sure how that's going